app/controllers/owners_controller.rb

```diff
@@ -10,19 +10,8 @@ def show
     # Only do this once
     session[:new_supporter] = false if @new_supporter
 
-    @scrapers = @owner.scrapers.includes(:last_run)
-
-    # Split out scrapers into different groups
-    @running_scrapers, @erroring_scrapers, @other_scrapers = [], [], []
-    @scrapers.each do |scraper|
-      if scraper.running?
-        @running_scrapers << scraper
-      elsif scraper.requires_attention?
-        @erroring_scrapers << scraper
-      else
-        @other_scrapers << scraper
-      end
-    end
+    @scrapers = @owner.scrapers.order(:updated_at).includes(:last_run)
```

Comment:
It's not so much that change was required by the issue. It's more that once I implemented the changes that were required by the issue, we were cycling through three object collections (@running_scrapers, @erroring_scrapers, @other_scrapers) in three identical ways in the one view. If you wanted to list the non-erroring scrapers first we could still achieve this with a sort on the one object collection (@scrapers).